modules/ROOT/pages/update-partner-profile.adoc

```diff
@@ -0,0 +1,26 @@
+= Update a Partner's Profile
+
```

Comment:
Maybe a snippet about under what circumstances an audience might want or need to do this?